plugins/aap-backend/README.md

```diff
@@ -1,24 +1,22 @@
-# AAP Backstage provider
+# Ansible Automation Platform Backstage provider plugin
 
-The AAP Backstage provider plugin synchronizes the AAP job templates and workflow job templates into the [Backstage](https://backstage.io/) catalog.
+The Ansible Automation Platform (AAP) Backstage provider plugin synchronizes the AAP content including job templates into the [Backstage](https://backstage.io/) catalog.
```

Comment:
we show job templates and workflow job templates in catalog if we want to mention this explictly 

this was added with https://github.com/janus-idp/backstage-plugins/pull/674

Reply:
Done